### PR TITLE
Fix the style of product reference in the stock management page

### PR DESCRIPTION
**Pull request by Azouz-Jribi**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.2.x
| Description?  | When creating a product with a reference longer than 24 characters, we have a bad display in stock page.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-3489
| How to test?  | Create a product reference with more than 24 characters, go to BO > Catalog > Stock and check in reference column that it is displayed correctly (in two lines).

### CODE REVIEW

File: admin-dev/themes/new-theme/public/theme.css [hunk 1]
```diff
@@ -12747,3 +12747,8 @@ span.twitter-typeahead {
   width: 100% !important; }
   span.twitter-typeahead input.tt-input {
     width: 100% !important; }
+
+.stock-app table td {
```
Comment (aleeks) [inline admin-dev/themes/new-theme/public/theme.css]: Hello @Azouz-Jribi 
It's not here to change the css to the vue application. It's directly in the vue file. 
@nihco2, can you explain better please!
Regards

**Review by aleeks — CHANGES_REQUESTED** [review]

File: admin-dev/themes/new-theme/public/theme.css (hunk 1 above)
Comment (Azouz-Jribi) [reply admin-dev/themes/new-theme/public/theme.css]: Hi @aleeks 
I know ;) 
But when i generate the assets, the "theme.css" file is modified.

Regards

**Comment by Azouz-Jribi** [comment]
Hello @aleeks 
Thank you for the information, I was not aware with the vue file.
Correction is done.

Regards

**Review by aleeks — DISMISSED** [review]

**Comment by aleeks** [comment]
Thank you @Azouz-Jribi 
Maybe, you will have to rebase when yours others PR will be merged :smile:

**Review by eternoendless — APPROVED** [review]

**Comment by aleeks** [comment]
Thank you @Azouz-Jribi

**Pull request merged**